FuelSDK-CSharp/ETClient.cs

```diff
@@ -91,8 +99,11 @@ public ETClient(NameValueCollection parameters = null, RefreshState refreshState
                 {
                     configSection.RestEndPoint = parameters["restEndPoint"];
                 }
+                if (parameters.AllKeys.Contains("useOAuth2Authentication"))
+                {
+                    configSection.UseOAuth2Authentication = parameters["useOAuth2Authentication"];
```

Comment:
@ncorn this is where we are getting the values from the parameter collection and assigning it to the configsection class.

Reply:
I'm not sure why I was overlooking it. Thanks for pointing it out.